sexpr/lexer.go

```diff
@@ -0,0 +1,64 @@
+package sexpr
+
+type Token int
+
+const (
+	EOF Token = iota
+	WHITESPACE
+	LPAREN // (
+	RPAREN // )
+	STRING
+	NUMBER
+	BOOL
+	COMMENT
+	SYMBOL
+)
+
+// isWhitespace returns true if the rune is the first rune of a
+// Whitespace sequence.
+func isWhitespace(r rune) bool {
+	return r == ' ' || r == '\t' || r == '\r' || r == '\n'
```

Comment:
:tulip: The unicode package provides a [`IsSpace` function](https://golang.org/pkg/unicode/#IsSpace) that deals with additional special characters. Do you think we should use it?

Reply:
Yes!  This was buzzing in my head on Friday - I knew I'd seen it somewhere.